build.gradle.kts

```diff
@@ -17,6 +19,31 @@ springBoot {
   mainClass = "com.mycompany.myapp.JhipsterSampleApplicationApp"
 }
 
+node {
+  version.set("v20.12.2")
+  npmVersion.set("10.5.2")
+}
+
+val testTaskUsingNpm = tasks.register<NpmTask>("testNpm") {
+  dependsOn("test", "integrationTest")
+  dependsOn(tasks.npmInstall)
+  npmCommand.set(listOf("run", "test"))
+//  args.set(listOf("test"))
+  ignoreExitValue.set(false)
+  environment.set(mapOf("APP_VERSION" to "0.0.1-SNAPSHOT")) //gradle doesnt use project version like maven. Where should I get that?
```

Comment:
I need help here: 
```kts
 environment.set(mapOf("APP_VERSION" to "0.0.1-SNAPSHOT")) //gradle doesnt use project version like maven. Where should I get that?
 ```

Reply:
It's the version defined here: https://github.com/renanfranca/sample-frontend-gradle/blob/45f55a752bb2e01177f08a820d1098f0e313ac71/build.gradle.kts#L28C1-L28C8
You can maybe access it with `project.version`

Reply:
Thank you!